integration/gethnetwork/build_geth_binary.sh

```diff
@@ -48,6 +48,12 @@ fi
 # Make sure .build folder exists
 mkdir -p "${build_path}"
 
+if [ -f "${geth_path}/geth-${geth_version}" ]
+then
+    echo "Skipping geth build - Found binary at ${geth_path}/geth-${geth_version}"
+    exit
```

Comment:
```suggestion
    exit 0
```

I think needs to be exit 0